commit 0bc66ba12806511bc416ae27779e104b2892b90f
Author: Narrat
Date:   Sat Jul 23 23:25:09 2016 +0200

    add mpd, remove wireless_tools and nvidia

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index c9e09d6..ad65624 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -11,8 +11,8 @@ url='https://github.com/brndnmtthws/conky'
 license=('BSD' 'GPL')
 arch=('x86_64')
 makedepends=('cmake' 'docbook2x' 'docbook-xsl' 'man-db' 'git')
-depends=('glib2' 'lua' 'wireless_tools' 'libxdamage' 'libxinerama' 'libxft'
-         'imlib2' 'libxml2' 'libpulse' 'libxnvctrl')
+depends=('glib2' 'lua' 'libxdamage' 'libxinerama' 'libxft'
+         'imlib2' 'libxml2' 'libpulse' 'mpd')
 source=("https://github.com/brndnmtthws/conky/archive/v${pkgver}.tar.gz")
 sha256sums=('4cefdd92219a90934c28297e4ac7448a3f69d6aeec5d48c5763b23f6b214ef13')
 options=('!strip' 'debug')
@@ -24,16 +24,17 @@ build() {
 		-D CMAKE_BUILD_TYPE=Release \
 		-D MAINTAINER_MODE=ON \
 		-D BUILD_DOCS=ON \
-		-D BUILD_WLAN=ON \
+		-D BUILD_WLAN=OFF \
 		-D BUILD_XDBE=ON \
 		-D BUILD_XSHAPE=ON \
 		-D BUILD_IMLIB2=ON \
 		-D BUILD_CURL=ON \
 		-D BUILD_RSS=ON \
-		-D BUILD_NVIDIA=ON \
+		-D BUILD_NVIDIA=OFF \
 		-D BUILD_WEATHER_METAR=ON \
 		-D BUILD_PULSEAUDIO=ON \
 		-D BUILD_JOURNAL=ON \
+		-D BUILD_MPD=ON \
 		-D CMAKE_INSTALL_PREFIX=/usr \
 		.
 
