commit b29246c2570aaef07269e9b637a90a4b8661da20
Author: Narrat
Date:   Tue Jul 26 01:50:11 2016 +0200

    add mpd to conky

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 15e7814..638f241 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -13,7 +13,7 @@ license=('BSD' 'GPL')
 arch=('i686' 'x86_64')
 makedepends=('cmake' 'docbook2x' 'docbook-xml' 'man-db' 'git')
 depends=('glib2' 'lua' 'wireless_tools' 'libxdamage' 'libxinerama' 'libxft'
-         'imlib2' 'libxml2' 'libpulse')
+         'imlib2' 'libxml2' 'libpulse' 'mpd')
 source=("https://github.com/brndnmtthws/conky/archive/v${pkgver}.tar.gz"
         'lua53.patch')
 sha256sums=('4c80982960b2a72e3e33cef33347c97b9239ed110d9584da4e7f17f28faf60ca'
@@ -41,6 +41,7 @@ build() {
 		-D BUILD_WEATHER_XOAP=ON \
 		-D BUILD_PULSEAUDIO=ON \
 		-D BUILD_JOURNAL=ON \
+		-D BUILD_MPD=ON \
 		-D CMAKE_INSTALL_PREFIX=/usr \
 		.
 
