commit 104bfe0677ab61d233381768df82ddc7203804f6
Author: Narrat
Date:   Tue Jul 26 01:50:11 2016 +0200

    add mpd to conky

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 33f4e7f..b446024 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -13,7 +13,7 @@ license=('BSD' 'GPL')
 arch=('i686' 'x86_64')
 makedepends=('cmake' 'docbook2x' 'docbook-xml' 'man-db' 'git')
 depends=('glib2' 'lua' 'wireless_tools' 'libxdamage' 'libxinerama' 'libxft'
-         'imlib2' 'libxml2' 'libpulse')
+         'imlib2' 'libxml2' 'libpulse' 'mpd')
 source=("https://github.com/brndnmtthws/conky/archive/v${pkgver}.tar.gz"
         'lua53.patch')
 sha1sums=('971b20be2d8c2e676374441469e8525708500776'
@@ -40,6 +40,7 @@ build() {
 		-D BUILD_WEATHER_METAR=ON \
 		-D BUILD_WEATHER_XOAP=ON \
 		-D BUILD_PULSEAUDIO=ON \
+		-D BUILD_MPD=ON \
 		-D CMAKE_INSTALL_PREFIX=/usr \
 		.
 
