commit 7a55af9886c41e52f34a8a7a3632a802e5055378
Author: Narrat
Date:   Sun Nov 6 17:53:24 2016 +0100

    Remove qt deps

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index cc6fa47..e4db54f 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -10,8 +10,8 @@ pkgdesc="Plotting package which outputs to X11, PostScript, PNG, GIF, and others
 arch=('i686' 'x86_64') 
 url="http://www.gnuplot.info" 
 license=('custom') 
-depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'qt5-svg' 'gnutls') 
-makedepends=('texinfo' 'texlive-core' 'emacs' 'texlive-latexextra' 'qt5-tools')
+depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'gnutls')
+makedepends=('texinfo' 'texlive-core' 'emacs' 'texlive-latexextra')
 options=('!makeflags')
 source=("http://downloads.sourceforge.net/sourceforge/$pkgname/$pkgname-$pkgver.tar.gz"
         "lua53_compat.patch")
@@ -37,7 +37,8 @@ build() {
   WX_CONFIG=/usr/bin/wx-config ./configure --prefix=/usr \
               --libexecdir=/usr/bin \
               --with-gihdir=/usr/share/gnuplot \
-              --with-readline=gnu --enable-qt \
+              --with-readline=gnu \
+              --with-qt=no \
               --with-bitmap-terminals
   make pkglibexecdir=/usr/bin
 }
