commit 532865d0aa4a3d8b477f5a5d9c1776b8eca090ae
Author: Narrat
Date:   Fri Oct 2 19:45:06 2015 +0200

    Remove qt deps

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 2eba58f..e585d2f 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -10,7 +10,7 @@ pkgdesc="Plotting package which outputs to X11, PostScript, PNG, GIF, and others
 arch=('i686' 'x86_64') 
 url="http://www.gnuplot.info" 
 license=('custom') 
-depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'qt5-svg' 'qt5-tools' 'gnutls') 
+depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'gnutls')
 makedepends=('texinfo' 'texlive-core' 'emacs' 'texlive-latexextra')
 options=('!makeflags')
 install=gnuplot.install
@@ -38,7 +38,8 @@ build() {
   WX_CONFIG=/usr/bin/wx-config ./configure --prefix=/usr \
               --libexecdir=/usr/bin \
               --with-gihdir=/usr/share/gnuplot \
-              --with-readline=gnu --enable-qt
+              --with-readline=gnu \
+              --with-qt=no
   make pkglibexecdir=/usr/bin
 }
 
