commit 4de25c9f87254d50e91bd4be5353220f277fb55c
Author: Narrat
Date:   Fri Jul 24 13:29:23 2015 +0200

    Remove qt deps

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index d4befc9..7277044 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -10,7 +10,7 @@ pkgdesc="Plotting package which outputs to X11, PostScript, PNG, GIF, and others
 arch=('i686' 'x86_64') 
 url="http://www.gnuplot.info" 
 license=('custom') 
-depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'qt5-svg' 'qt5-tools' 'gnutls') 
+depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'gnutls')
 makedepends=('texinfo' 'texlive-core' 'emacs' 'texlive-latexextra')
 options=('!makeflags')
 install=gnuplot.install
@@ -38,7 +38,7 @@ build() {
   WX_CONFIG=/usr/bin/wx-config ./configure --prefix=/usr \
               --libexecdir=/usr/bin \
               --with-gihdir=/usr/share/gnuplot \
-              --with-readline=gnu --enable-qt
+              --with-readline=gnu
   make pkglibexecdir=/usr/bin
 }
 
