commit b0638fdbfbc7c831f7f99bd2413532b055086bb9
Author: Narrat
Date:   Thu Apr 21 18:05:11 2016 +0200

    Remove qt deps

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 1e99f85..45e5d42 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -10,7 +10,7 @@ pkgdesc="Plotting package which outputs to X11, PostScript, PNG, GIF, and others
 arch=('i686' 'x86_64') 
 url="http://www.gnuplot.info" 
 license=('custom') 
-depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'qt5-svg' 'qt5-tools' 'gnutls') 
+depends=('readline' 'gd' 'wxgtk' 'cairo' 'libjpeg' 'lua' 'gnutls')
 makedepends=('texinfo' 'texlive-core' 'emacs' 'texlive-latexextra')
 options=('!makeflags')
 install=gnuplot.install
@@ -38,7 +38,8 @@ build() {
   WX_CONFIG=/usr/bin/wx-config ./configure --prefix=/usr \
               --libexecdir=/usr/bin \
               --with-gihdir=/usr/share/gnuplot \
-              --with-readline=gnu --enable-qt \
+              --with-readline=gnu \
+              --with-qt=no \
               --with-bitmap-terminals
   make pkglibexecdir=/usr/bin
 }
