commit d7ea240abb493fa576024676d6ced46677ce9461
Author: Narrat <autumn-wind@web.de>
Date:   Thu Jan 19 22:58:39 2017 +0100

    Remove QT deps and disable build of py2 version

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 4a48d1c..20c2435 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -13,25 +13,22 @@ pkgdesc="A python plotting library, making publication quality plots"
 arch=('x86_64')
 url='http://matplotlib.org'
 license=('custom')
-checkdepends=('python-pytest-xdist' 'python2-pytest-xdist' 'python-pytest-runner'
-              'python2-pytest-runner' 'python2-mock' 'xorg-server-xvfb' 'texlive-core'
-              'texlive-latexextra' 'imagemagick' 'ffmpeg' 'mencoder' 'inkscape' 'python-pandas'
-              'python2-pandas' 'ttf-freefont')
-makedepends=('python2-pytz' 'python2-numpy' 'python2-pyqt4' 'python-pytz' 'python-numpy'
-             'python-pyqt4' 'tk' 'python-cairocffi' 'python2-cairocffi' 'python-dateutil'
-             'python2-dateutil' 'python-gobject' 'python2-gobject' 'python-pyparsing'
-             'python2-pyparsing' 'pygtk' 'python-six' 'ghostscript' 'texlive-bin'
-             'python-tornado' 'python2-tornado' 'gtk3' 'wxpython' 'python-pyqt5' 'python2-pyqt5'
-             'libxkbcommon-x11' 'python-pillow' 'python2-pillow' 'python-setuptools'
-             'python2-setuptools' 'python-cycler' 'python2-cycler' 'python-kiwisolver'
-             'python2-kiwisolver')
+checkdepends=('python-pytest-xdist' 'python-pytest-runner' 'xorg-server-xvfb' 'texlive-core'
+              'texlive-latexextra' 'imagemagick' 'ffmpeg' 'mencoder' 'inkscape' 'python-pandas' 'ttf-freefont')
+#checkdepends+=('python2-pytest-xdist' 'python2-pytest-runner' 'python2-mock' 'python2-pandas')
+makedepends=('python-pytz' 'python-numpy' 'tk' 'python-cairocffi' 'python-dateutil'
+             'python-gobject' 'python-pyparsing' 'python-six' 'ghostscript' 'texlive-bin'
+             'python-tornado' 'gtk3' 'libxkbcommon-x11' 'python-pillow' 'python-setuptools' 'python-cycler' 'python-kiwisolver')
+# Python2 deps
+#makedepends+=('python2-pytz' 'python2-numpy' 'python2-cairocffi' 'python2-gobject'
+#            'python2-dateutil' 'python2-pyparsing' 'pygtk' 'python2-tornado' 'python2-pillow' 'python2-setuptools' 'python2-cycler' 'wxpython' 'python2-kiwisolver')
 source=("$pkgbase-$pkgver.tar.gz::https://github.com/matplotlib/matplotlib/archive/v$pkgver.tar.gz"
         setup.cfg)
 sha512sums=('58dbeabafa7cd008cf5a585829a5b9f098114a43700dedbd7cc3ed09180ea99a870d06295fe8f1d47f1eb807d264924a4b6fb7cf47e6bc3b74045aa4f0714ac9'
             'f08c0b2e94599fdf2b736b8a655d862209934441d90f20ed872cfc128b8d2a6525763ab7cf64f2b3630e22a90798f4c52aa93b18087e9b48708d9bae34aada0b')
 
 prepare() {
-  cp -a "$srcdir"/matplotlib-$pkgver{,-py2}
+  #cp -a "$srcdir"/matplotlib-$pkgver{,-py2}
 
   cd "$srcdir"/matplotlib-$pkgver
   for file in $(find . -name '*.py' -print); do
@@ -39,18 +36,18 @@ prepare() {
            -e "s|^#!.*/usr/bin/env *python|#!/usr/bin/env python3|" $file
   done
 
-  cd "$srcdir"/matplotlib-$pkgver-py2
-  for file in $(find . -name '*.py' -print); do
-    sed -i -e "s|^#!.*/usr/bin/python|#!/usr/bin/python2|" \
-           -e "s|^#!.*/usr/bin/env *python|#!/usr/bin/env python2|" $file
-  done
+  #cd "$srcdir"/matplotlib-$pkgver-py2
+  #for file in $(find . -name '*.py' -print); do
+    #sed -i -e "s|^#!.*/usr/bin/python|#!/usr/bin/python2|" \
+      #     -e "s|^#!.*/usr/bin/env *python|#!/usr/bin/env python2|" $file
+  #done
 
   cp -a "$srcdir"/matplotlib-$pkgver{,-test}
-  cp -a "$srcdir"/matplotlib-$pkgver-py2{,-test}
+  #cp -a "$srcdir"/matplotlib-$pkgver-py2{,-test}
 
   # Configure tests (FS#48175)
   cp "$srcdir/setup.cfg" "$srcdir"/matplotlib-$pkgver-test/
-  cp "$srcdir/setup.cfg" "$srcdir"/matplotlib-$pkgver-py2-test/
+  #cp "$srcdir/setup.cfg" "$srcdir"/matplotlib-$pkgver-py2-test/
 }
 
 build() {
@@ -61,8 +58,8 @@ build() {
   cd "$srcdir"/matplotlib-$pkgver
   python3 setup.py build
 
-  cd "$srcdir"/matplotlib-$pkgver-py2
-  python2 setup.py build
+  #cd "$srcdir"/matplotlib-$pkgver-py2
+  #python2 setup.py build
 }
 
 check() {
@@ -72,17 +69,16 @@ check() {
   xvfb-run -a -s "+extension GLX +extension RANDR +render -screen 0 1280x1024x24" \
     python setup.py pytest --addopts="-n auto" || warning "Tests failed"
 
-  cd "$srcdir"/matplotlib-$pkgver-py2-test
-  xvfb-run -a -s "+extension GLX +extension RANDR +render -screen 0 1280x1024x24" \
-    python2 setup.py pytest --addopts="-n auto" || warning "Tests failed"
+  #cd "$srcdir"/matplotlib-$pkgver-py2-test
+  #xvfb-run -a -s "+extension GLX +extension RANDR +render -screen 0 1280x1024x24" \
+    #python2 setup.py pytest --addopts="-n auto" || warning "Tests failed"
 }
 
 package_python-matplotlib() {
-  depends=('python-pytz' 'python-numpy' 'python-pyqt5' 'python-dateutil' 'python-pyparsing'
+  depends=('python-pytz' 'python-numpy' 'python-dateutil' 'python-pyparsing'
            'python-cycler' 'python-kiwisolver' 'libxkbcommon-x11')
   optdepends=('python-gobject: for GTK3Agg/GTK3Cairo backend'
               'python-cairocffi: for GTK3Agg/GTK3Cairo backend'
-              'python-pyqt4: for Qt4Agg backend'
               'tk: used by the TkAgg backend'
               'ghostscript: usetex dependencies'
               'texlive-bin: usetex dependencies'
@@ -97,23 +93,24 @@ package_python-matplotlib() {
 }
 
 package_python2-matplotlib() {
-  depends=('python2-pytz' 'python2-numpy' 'python2-pyqt5' 'python2-dateutil' 'python2-pyparsing'
-           'python2-cycler' 'python2-kiwisolver' 'libxkbcommon-x11')
-  optdepends=('pygtk: for GTK/GTKAgg/GTKCairo backend'
-              'python2-cairo: for GTKCairo/GTK3Cairo backend'
-              'python2-cairocffi: for GTKCairo/GTK3Cairo backend (alternative to python2-cairo)'
-              'python2-pyqt4: for Qt4Agg backend'
-              'tk: used by the TkAgg backend'
-              'ghostscript: usetex dependencies'
-              'texlive-bin: usetex dependencies'
-              'python2-tornado: for webagg backend'
-              'python2-gobject: for GTK3Agg/GTK3Cairo backend'
-              'wxpython: for WX/WXAgg backend'
-              'python2-pillow: for reading/saving .jpg/bmp/tiff files')
+  #depends=('python2-pytz' 'python2-numpy' 'python2-dateutil' 'python2-pyparsing'
+    #       'python2-cycler' 'python2-kiwisolver' 'libxkbcommon-x11')
+  #optdepends=('pygtk: for GTK/GTKAgg/GTKCairo backend'
+    #          'python2-cairo: for GTKCairo/GTK3Cairo backend'
+    #          'python2-cairocffi: for GTKCairo/GTK3Cairo backend (alternative to python2-cairo)'
+    #          'python2-pyqt4: for Qt4Agg backend'
+    #          'tk: used by the TkAgg backend'
+    #          'ghostscript: usetex dependencies'
+    #          'texlive-bin: usetex dependencies'
+    #          'python2-tornado: for webagg backend'
+    #          'python2-gobject: for GTK3Agg/GTK3Cairo backend'
+    #          'wxpython: for WX/WXAgg backend'
+    #          'python2-pillow: for reading/saving .jpg/bmp/tiff files')
 
-  cd matplotlib-$pkgver-py2
-  python2 setup.py install -O1 --skip-build --root "$pkgdir" --prefix=/usr
+  #cd matplotlib-$pkgver-py2
+  #python2 setup.py install -O1 --skip-build --root "$pkgdir" --prefix=/usr
 
-  install -dm755 "$pkgdir"/usr/share/licenses/python2-matplotlib
-  install -m 644 doc/users/license.rst "$pkgdir"/usr/share/licenses/python2-matplotlib
+  #install -dm755 "$pkgdir"/usr/share/licenses/python2-matplotlib
+  #install -m 644 doc/users/license.rst "$pkgdir"/usr/share/licenses/python2-matplotlib
+  msg2 "Nothing to see"
 }
