commit 2c5f9d5674a0122e81ed830376458e9b5b543587
Author: Narrat
Date:   Sun Feb 24 14:51:23 2019 +0100

    remove smbclient from deps

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 7601421..c80aa14 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -15,7 +15,7 @@ depends=('audiofile' 'avahi' 'curl' 'faad2' 'ffmpeg' 'fluidsynth' 'icu' 'jack'
          'libao' 'libcdio-paranoia' 'libgme' 'libid3tag' 'libmad' 'libmikmod'
          'libmms' 'libmodplug' 'libmpcdec' 'libmpdclient' 'libnfs'
          'libsamplerate' 'libshout' 'libsoxr' 'systemd-libs' 'libupnp' 'mpg123'
-         'openal' 'smbclient' 'sqlite' 'twolame' 'wavpack' 'wildmidi' 'yajl'
+         'openal' 'sqlite' 'twolame' 'wavpack' 'wildmidi' 'yajl'
          'zziplib')
 makedepends=('boost' 'meson' 'python-sphinx')
 validpgpkeys=('0392335A78083894A4301C43236E8A58C6DB4512')
@@ -47,6 +47,7 @@ build() {
 	       '-Dadplug=disabled' # not in an official repo
 	       '-Dsndio=disabled' # interferes with detection of alsa devices
 	       '-Dshine=disabled' # not in an official repo
+	       '-Dsmbclient=disabled' # do not want
 	)
 	arch-meson .. ${_opts[@]}
 	ninja
