commit e570966341ffdd92295404b85262723dfffc6331
Author: Narrat
Date:   Sat Nov 21 19:20:11 2020 +0100

    Remove smbclient from deps

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index ddbb428..1fe92ca 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -22,7 +22,7 @@ depends=('alsa-lib' 'libasound.so' 'desktop-file-utils' 'ffmpeg' 'libavcodec.so'
          'libva' 'libva.so' 'libva-drm.so' 'libva-wayland.so' 'libva-x11.so' 'libvdpau' 'libx11'
          'libxext' 'libxinerama' 'libxkbcommon' 'libxkbcommon.so' 'libxrandr' 'libxss' 'libxss'
          'libxv' 'lua52' 'mesa' 'mujs' 'rubberband' 'librubberband.so' 'shaderc'
-         'libshaderc_shared.so' 'smbclient' 'uchardet' 'vulkan-icd-loader' 'wayland' 'xdg-utils'
+         'libshaderc_shared.so' 'uchardet' 'vulkan-icd-loader' 'wayland' 'xdg-utils'
          'zlib')
 makedepends=('git' 'mesa' 'python-docutils' 'ladspa' 'wayland-protocols'
              'ffnvcodec-headers' 'vulkan-headers' 'waf')
@@ -43,7 +43,7 @@ build() {
     --enable-dvdnav \
     --enable-libarchive \
     --enable-libmpv-shared \
-    --enable-libsmbclient \
+    --disable-libsmbclient \
     --disable-build-date
 
   waf build
