commit 85a58b225bcc4b42af4be693ed960c65c95a53ae
Author: Narrat
Date:   Wed Jun 10 17:16:26 2015 +0200

    Disable samba and libcaca support

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 1b5c5ae..db57934 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -12,9 +12,9 @@ license=('GPL')
 url='http://mpv.io'
 depends=(
   'ffmpeg' 'lcms2' 'libcdio-paranoia' 'libgl' 'enca' 'libxss'
-  'libxinerama' 'libxv' 'libxkbcommon' 'libva' 'wayland' 'libcaca'
+  'libxinerama' 'libxv' 'libxkbcommon' 'libva' 'wayland'
   'desktop-file-utils' 'hicolor-icon-theme' 'xdg-utils' 'lua52' 'libdvdnav'
-  'libguess' 'libxrandr' 'jack' 'smbclient' 'rubberband'
+  'libguess' 'libxrandr' 'jack' 'rubberband'
 )
 makedepends=('mesa' 'python-docutils' 'ladspa' 'hardening-wrapper')
 optdepends=('youtube-dl: for video-sharing websites playback')
