commit 979aba54bc077116c41c858756bee6e4d9a4ca22
Author: Narrat <autumn-wind@web.de>
Date:   Fri Dec 15 19:23:47 2017 +0100

    Disable samba support

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 6787f285..dc85a07f 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -18,7 +18,7 @@ depends=(
   'ffmpeg' 'lcms2' 'libcdio-paranoia' 'libgl' 'libxss'
   'libxinerama' 'libxv' 'libxkbcommon' 'libva' 'wayland' 'libcaca'
   'desktop-file-utils' 'hicolor-icon-theme' 'xdg-utils' 'lua52' 'libdvdnav'
-  'libxrandr' 'jack' 'rubberband' 'uchardet' 'libarchive' 'smbclient'
+  'libxrandr' 'jack' 'rubberband' 'uchardet' 'libarchive'
 )
 makedepends=('mesa' 'python-docutils' 'ladspa')
 optdepends=('youtube-dl: for video-sharing websites playback')
@@ -60,7 +60,6 @@ build() {
     --enable-encoding \
     --enable-libarchive \
     --enable-libmpv-shared \
-    --enable-libsmbclient \
     --enable-zsh-comp
 
   ./waf build
