commit 977ff1a14136a1a77fcc4672f86829f145f18bac
Author: Narrat
Date:   Sat Feb 27 17:51:31 2016 +0100

    Disable samba support

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index e77d6ce..9b14af9 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -15,7 +15,7 @@ depends=(
   'ffmpeg' 'lcms2' 'libcdio-paranoia' 'libgl' 'enca' 'libxss'
   'libxinerama' 'libxv' 'libxkbcommon' 'libva' 'wayland' 'libcaca'
   'desktop-file-utils' 'hicolor-icon-theme' 'xdg-utils' 'lua52' 'libdvdnav'
-  'libxrandr' 'jack' 'smbclient' 'rubberband'
+  'libxrandr' 'jack' 'rubberband'
 )
 makedepends=('mesa' 'python-docutils' 'ladspa' 'hardening-wrapper')
 optdepends=('youtube-dl: for video-sharing websites playback')
