commit 83e9a3e61c63bb0d49927681ff4b76fd78ea0e65
Author: Narrat
Date:   Fri Oct 2 19:41:27 2015 +0200

    Disable samba support

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 443e270..890f88c 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -15,7 +15,7 @@ depends=(
   'ffmpeg' 'lcms2' 'libcdio-paranoia' 'libgl' 'enca' 'libxss'
   'libxinerama' 'libxv' 'libxkbcommon' 'libva' 'wayland' 'libcaca'
   'desktop-file-utils' 'hicolor-icon-theme' 'xdg-utils' 'lua52' 'libdvdnav'
-  'libxrandr' 'jack' 'smbclient' 'rubberband'
+  libxrandr' 'jack' 'rubberband'
 )
 makedepends=('mesa' 'python-docutils' 'ladspa' 'hardening-wrapper')
 optdepends=('youtube-dl: for video-sharing websites playback')
