commit 99cb82e00f0572e6c5d27eaeb7ffcf2f3223ad83
Author: Narrat
Date:   Sat May 14 11:10:21 2016 +0200

    Remove bluez-libs

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 32929be..0c32620 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -16,7 +16,7 @@ arch=('i686' 'x86_64')
 license=('GPL2' 'LGPL2.1')
 url='http://wiki.qemu.org/'
 depends=('pixman' 'libjpeg' 'libpng' 'sdl2' 'alsa-lib' 'nss' 'glib2'
-         'gnutls' 'bluez-libs' 'vde2' 'util-linux' 'libsasl' 'libgl'
+         'gnutls' 'vde2' 'util-linux' 'libsasl' 'libgl'
          'seabios' 'libcap' 'libcap-ng' 'libaio' 'libseccomp' 'libcacard'
          'spice' 'usbredir' 'lzo' 'snappy' 'gcc-libs' 'zlib' 'bzip2' 'nspr'
          'ncurses' 'libx11' 'libusb' 'libpulse' 'libssh2' 'curl' 'vte3'
