commit 5bd914db9353d606ae9cdd862e9dea0e729cb5dc
Author: Narrat <autumn-wind@web.de>
Date:   Wed Sep 27 01:15:04 2017 +0200

    Add ctrl for opening links

diff --git a/trunk/0002_additional_button_for_url.diff b/trunk/0002_additional_button_for_url.diff
new file mode 100644
index 0000000..4fbc064
--- /dev/null
+++ b/trunk/0002_additional_button_for_url.diff
@@ -0,0 +1,12 @@
+--- a/termite.cc
++++ b/termite.cc
+@@ -1108,7 +1108,7 @@
+ }
+ 
+ gboolean button_press_cb(VteTerminal *vte, GdkEventButton *event, const config_info *info) {
+-    if (info->clickable_url && event->type == GDK_BUTTON_PRESS) {
++    if (info->clickable_url && event->type == GDK_BUTTON_PRESS && (event->state & GDK_CONTROL_MASK)) {
+         auto match = make_unique(check_match(vte, event), g_free);
+         if (!match)
+             return FALSE;
+
diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index a39b006..5d7c517 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -8,8 +8,17 @@ arch=('i686' 'x86_64')
 url="https://github.com/thestinger/termite/"
 license=('LGPL')
 makedepends=('git' 'vte3-ng' 'ncurses')
-source=("git+https://github.com/thestinger/termite#tag=v$pkgver")
-md5sums=('SKIP')
+source=("git+https://github.com/thestinger/termite#tag=v$pkgver"
+        "0002_additional_button_for_url.diff")
+md5sums=('SKIP'
+         '6eab9b86b5f89175b09975fe4744faa8')
+
+
+prepare() {
+  cd $pkgname
+  # Add additional button for opening links
+  patch -Np1 -i $srcdir/0002_additional_button_for_url.diff
+}
 
 build() {
   cd $pkgbase
