commit 6e7b76d21cbe8f67a1e2019f42eb976601e9bb96
Author: Narrat <autumn-wind@web.de>
Date:   Sun Feb 14 02:25:15 2016 +0200

    Add ctrl to opening links

diff --git a/trunk/0002_additional_button_for_url.diff b/trunk/0002_additional_button_for_url.diff
new file mode 100644
index 0000000..c29e797
--- /dev/null
+++ b/trunk/0002_additional_button_for_url.diff
@@ -0,0 +1,11 @@
+--- a/termite.cc
++++ a/termite.cc
+@@ -1022,7 +1022,7 @@
+ }
+ 
+ gboolean button_press_cb(VteTerminal *vte, GdkEventButton *event, const config_info *info) {
+-    if (info->clickable_url && event->type == GDK_BUTTON_PRESS) {
++    if (info->clickable_url && event->type == GDK_BUTTON_PRESS && (event->state & GDK_CONTROL_MASK)) {
+         auto match = make_unique(check_match(vte, (int)event->x, (int)event->y), g_free);
+         if (!match)
+             return FALSE;
diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 64bbb16..d660160 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -11,8 +11,17 @@ license=('LGPL')
 depends=('vte3-ng')
 makedepends=('git')
 backup=(etc/xdg/termite/config)
-source=("git://github.com/thestinger/termite#tag=v$pkgver")
-md5sums=('SKIP')
+source=("git://github.com/thestinger/termite#tag=v$pkgver"
+        "0002_additional_button_for_url.diff")
+md5sums=('SKIP'
+         '6fc4a5a22a1d655e16641ae4cab130e2')
+
+
+prepare() {
+  cd $pkgname
+  # Add additional button for opening links
+  patch -Np1 -i $srcdir/0002_additional_button_for_url.diff
+}
 
 build() {
   cd $pkgname
