commit dd73171c79bda82156d58f1ba8b573ca7c3dd7de
Author: Narrat
Date:   Sun Jan 29 20:39:54 2017 +0100

    Moved the murrine engine into optdeps

diff --git a/PKGBUILD b/PKGBUILD
index dee85e3..6fef5fd 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,7 +10,7 @@ _releasever=20170128
 arch=('any')
 url='http://horst3180.deviantart.com/art/Vertex-Theme-470663601'
 license=('GPL3')
-depends=('gtk-engine-murrine')
+optdepends=('gtk-engine-murrine: For the GTK2 theme')
 conflicts=('vertex-themes-git')
 source=("${pkgname}-${_releasever}.tar.gz::https://github.com/horst3180/Vertex-theme/archive/${_releasever}.tar.gz")
 sha256sums=('1540657ff247bcdb9c49a740e4ddf305aecd4f3bebc93ca566fe74d319b7a620')
