commit fc24378064e05c05ac866084e575a7dedb5fdbdf
Author: Narrat
Date:   Sat Mar 19 17:51:12 2016 +0100

    zenity: disable webkitgtk support

diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 7d31ab1..f0846cb 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -8,7 +8,7 @@ pkgdesc="Display graphical dialog boxes from shell scripts"
 url="https://git.gnome.org/browse/zenity"
 arch=(i686 x86_64)
 license=(LGPL)
-depends=(webkit2gtk libnotify)
+depends=(libnotify)
 makedepends=(itstool yelp-tools gettext gnome-common git)
 _commit=cd1647cb2dfbc0e83ee372bee49028acbaf9a781  # master
 source=("git://git.gnome.org/zenity#commit=$_commit")
@@ -27,7 +27,8 @@ prepare() {
 build() {
   cd $pkgname
   ./configure --prefix=/usr --sysconfdir=/etc \
-      --localstatedir=/var
+      --localstatedir=/var \
+      --disable-webkitgtk
   make
 }
 
